Library/Homebrew/os/mac/xcode.rb

```diff
@@ -27,6 +27,11 @@ def latest_version
         end
       end
 
+      def prerelease?
+        # TODO: bump version after Xcode 8.0 is stable.
+        version > "7.3.1"
```

Comment:
I think once Xcode 8 is stable, we should bump this to `version >= "8.1"` as otherwise people will come complaining that `brew doctor` is unhappy when they update to 8.0.1 immediately (and before we get a chance to bump the version in here).

And we know already that anything `> "8.0"` and `< "8.1"` isn't going to be a beta release but a bugfix release, right?